modules/model_package/spleen_segmentation/programs/__init__.py

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
perhaps the name of this folder could be `scripts`

Reply:
Good point, updated in the latest commit.

Thanks.